rust/alloc/rc.rs

```diff
@@ -143,7 +143,7 @@
 //! ```
 //!
 //! If our requirements change, and we also need to be able to traverse from
-//! `Owner` to `Gadget`, we will run into problems. An [`Rc`] pointer from `Owner`
+//! `Owner` to `Gadget`, we will run into problems. An [`Rc`] pointer from `Owner`
```

Comment:
I honestly don't know what changed here

Reply:
This is a nbsp (0xA0) to space (0x20) change.